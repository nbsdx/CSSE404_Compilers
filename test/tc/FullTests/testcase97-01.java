class LightBulb {
	public static void main(String [] surrealists) {
		int giraffeHolder = 1;
		LightBulb BathtubFiller = new LightBulb();
		int howMany = giraffeHolder + BathtubFiller;
	}
}

