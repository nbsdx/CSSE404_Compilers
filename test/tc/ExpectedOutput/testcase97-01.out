Right argument of type LightBulb does not match expected type int for operator PLUS
